core-web/libs/portlets/edit-ema/ui/src/lib/dot-content-compare/dot-content-compare.component.ts

```diff
@@ -27,10 +20,11 @@ export class DotContentCompareComponent {
         }
     }
     @Output() shutdown = new EventEmitter<boolean>();
+    @Output() letMeBringBack = new EventEmitter<{ name: string; args: string[] }>();
```

Comment:
can it be just `bringBack`?

Reply:
The component already has a method called like this
I didn't want to change the existing code